constraint/pkg/apis/templates/v1alpha1/constrainttemplate_types_test.go

```diff
@@ -0,0 +1,55 @@
+/*
+
+Licensed under the Apache License, Version 2.0 (the "License");
+you may not use this file except in compliance with the License.
+You may obtain a copy of the License at
+
+    http://www.apache.org/licenses/LICENSE-2.0
+
+Unless required by applicable law or agreed to in writing, software
+distributed under the License is distributed on an "AS IS" BASIS,
+WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+See the License for the specific language governing permissions and
+limitations under the License.
+*/
+
+package v1alpha1
+
+import (
+	"testing"
+
+	"github.com/onsi/gomega"
+	"golang.org/x/net/context"
+	metav1 "k8s.io/apimachinery/pkg/apis/meta/v1"
+	"k8s.io/apimachinery/pkg/types"
+)
+
+func TestStorageConstraintTemplate(t *testing.T) {
+	key := types.NamespacedName{
```

Comment:
Is this code generated or is there some desire to use gomega? I'd prefer to just use the Go standard library for testing and not rely on one of these frameworks.

Reply:
This is generated code.

Reply:
OK. If let's merge for now and then revisit. If we don't need to take a dependency on gomega, that would be preferable.